File: app/src/androidTest/java/com/example/benchmarks/models/AppComponent.java
package com.example.benchmarks.models;

import com.example.benchmarks.ui.benchmark.BenchMarkViewModelFactory;

import javax.inject.Singleton;

import dagger.Component;

@Singleton
@Component(modules = TestBenchmarksModule.class)
public interface AppComponent {
    void inject(BenchMarkViewModelFactory benchMarkViewModelFactory);
}

File: app/src/androidTest/java/com/example/benchmarks/models/AppComponent.kt
package com.example.benchmarks.models

import javax.inject.Singleton
import com.example.benchmarks.ui.benchmark.BenchMarkViewModelFactory
import dagger.Component

@Singleton
@Component(modules = [TestBenchmarksModule::class])
interface AppComponent {
    fun inject(benchMarkViewModelFactory: BenchMarkViewModelFactory?)
}